docs/lib/content/configuring-npm/package-json.md

```diff
@@ -516,11 +516,9 @@ same shortcut syntax you use for `npm install`:
 }
 ```
 
-If the `package.json` for your package is not in the root directory (for
-example if it is part of a monorepo), you can specify the directory in
-which it lives:
-
-```json
+If your `package.json` isn't in the root directory, common with monorepos like Facebook's React, you can specify its location. i.e. `react-dom`:
+```jsonc
+/* @/packages/react-dom/package.json */
```

Comment:
I'm not sure this is a helpful comment, is it just a bare comment of a path with some sort of "shorthand" for `@` meaning `cwd`?

Reply:
> I'm not sure this is a helpful comment, is it just a bare comment of a path with some sort of "shorthand" for `@` meaning `cwd`?

Yea. I couldn't think of a good shorthand. This felt too long:
https://github.com/facebook/react/blob/main/packages/react-dom/package.json

I wanted to be verbose on which package.json the block was referring to